core/src/main/scala/org/apache/spark/api/java/JavaRDDLike.scala

```diff
@@ -368,6 +368,9 @@ trait JavaRDDLike[T, This <: JavaRDDLike[T, This]] extends Serializable {
   def toLocalIterator(): JIterator[T] =
      asJavaIteratorConverter(rdd.toLocalIterator).asJava
 
+  def toLocalIterator(prefetchPartitions: Boolean): JIterator[T] =
```

Comment:
nit: `prefetchPartitions` sounds like it will prefetch multiple partitions, but it is only the next one correct? Maybe rename this and elsewhere to `prefetchNextPartition`?

Reply:
I named if after @holdenk 's naming in Python. If you insist, I will rename